runner/runner/main.py

```diff
@@ -268,6 +270,7 @@ async def load_monado(config: Dict[str, Any]) -> None:
             ILLIXR_PATH=str(runtime_path / "runtime" / f"plugin.{profile}.so"),
             ILLIXR_COMP=":".join(map(str, plugin_paths)),
             ILLIXR_DATA=config["data"],
+            ILLIXR_DEMO_DATA=config["demo_data"],
```

Comment:
Remove. `monado.yaml` does not have demo_data because we don't run gldemo and debugview with Monado.

Reply:
I mean... It would be really nice if we can run debugview with monado in the future.